base/array.jl

```diff
@@ -177,14 +177,20 @@ false
 isbitsunion(u::Union) = allocatedinline(u)
 isbitsunion(x) = false
 
-function _unsetindex!(A::Array{T}, i::Int) where {T}
+@inbounds function _unsetindex!(A::Array{T}, i::Int) where {T}
```

Comment:
What does `@inbounds` on a function do?

Reply:
Supposed to be `@inline`.

Reply:
See https://github.com/JuliaLang/julia/pull/34127